bin/i18n/i18n_script_utils.rb

```diff
@@ -89,12 +89,13 @@ def redact(source, dest, *plugins)
 
   plugins = plugins_to_arg(plugins)
   data = YAML.load_file(source)
+  args = ['bin/i18n/node_modules/.bin/redact',
+          '-c bin/i18n/plugins/nonCommonmarkLinebreak.js']
+  unless plugins.empty?
+    args.push('-p ' + plugins)
+  end
```

Comment:
nit: this can be just `args.push('-p ' + plugins) unless plugins.empty?` all on one line

Reply:
Done.